modules/gojira-core/src/main/java/com/flipkart/gojira/core/ProfileRepository.java

```diff
@@ -56,6 +56,20 @@ protected String initialValue() {
         }
       };
   private static ProfileSetting globalProfileSetting = new ProfileSetting();
+
+  private static InheritableThreadLocal<ProfileSetting> threadLocalProfileSetting =
```

Comment:
variable name can be renamed to requestProfileSetting

Reply:
Done